### PR TITLE
Bump Django and ipython

### PR DESCRIPTION
**Pull request by abottoms-coder**
## Summary of Changes
_Provide a brief summary of changes_
Responding to dependabot having resolution failures in the security tab alerts for known patched vulnerabilities.

Django 3.2.11 -> 3.2.12
ipython 7.25.0 -> 7.31.1

### CODE REVIEW

**Review by raftmsohani — APPROVED** [review]
Ran with the new version. No issues

**Review by riatzukiza — APPROVED** [review]
Ran it, looks good

**Review by jorgegonzalez — APPROVED** [review]

File: tdrs-backend/Pipfile [hunk 1]
```diff
@@ -21,10 +21,11 @@ pytest-mock = "*"
 pytest-factoryboy = "*"
 
 [packages]
+setuptools = "<58.0"
```
Comment (ADPennington) [inline tdrs-backend/Pipfile:24]: @abottoms-coder what is this needed for?

Comment (abottoms-coder) [reply tdrs-backend/Pipfile:24]: The latest version of setuptools (60) gets rid of a function used by several of our dependencies including ipython.

**Review by ADPennington — APPROVED** [review]
backend spins up with no problems. 👍🏾

**Comment by abottoms-coder** [comment]
Need to investigate and resolve why this couldn't deploy:

```
2022-02-23T10:11:14.45-0500 [STG/0] ERR            creating build/lib/coreschema/templates
   2022-02-23T10:11:14.45-0500 [STG/0] ERR            copying coreschema/templates/base.html -> build/lib/coreschema/templates
   2022-02-23T10:11:14.45-0500 [STG/0] ERR            Traceback (most recent call last):
   2022-02-23T10:11:14.45-0500 [STG/0] ERR              File "<string>", line 1, in <module>
   2022-02-23T10:11:14.45-0500 [STG/0] ERR              File "/tmp/pip-install-mm1pdw5s/coreschema_1a3670c56bbb46f3a443e248793a3f3f/setup.py", line 55, in <module>
   2022-02-23T10:11:14.45-0500 [STG/0] ERR                setup(
   2022-02-23T10:11:14.45-0500 [STG/0] ERR              File "/tmp/contents3962303828/deps/0/python/lib/python3.10/site-packages/setuptools/__init__.py", line 153, in setup
   2022-02-23T10:11:14.45-0500 [STG/0] ERR                return distutils.core.setup(**attrs)
   2022-02-23T10:11:14.45-0500 [STG/0] ERR              File "/tmp/contents3962303828/deps/0/python/lib/python3.10/distutils/core.py", line 148, in setup
   2022-02-23T10:11:14.45-0500 [STG/0] ERR                dist.run_commands()
   2022-02-23T10:11:14.45-0500 [STG/0] ERR              File "/tmp/contents3962303828/deps/0/python/lib/python3.10/distutils/dist.py", line 966, in run_commands
   2022-02-23T10:11:14.45-0500 [STG/0] ERR                self.run_command(cmd)
   2022-02-23T10:11:14.45-0500 [STG/0] ERR              File "/tmp/contents3962303828/deps/0/python/lib/python3.10/distutils/dist.py", line 985, in run_command
   2022-02-23T10:11:14.45-0500 [STG/0] ERR                cmd_obj.run()
   2022-02-23T10:11:14.45-0500 [STG/0] ERR              File "/tmp/contents3962303828/deps/0/python/lib/python3.10/site-packages/setuptools/command/install.py", line 61, in run
   2022-02-23T10:11:14.45-0500 [STG/0] ERR                return orig.install.run(self)
   2022-02-23T10:11:14.45-0500 [STG/0] ERR              File "/tmp/contents3962303828/deps/0/python/lib/python3.10/distutils/command/install.py", line 568, in run
   2022-02-23T10:11:14.45-0500 [STG/0] ERR                self.run_command('build')
   2022-02-23T10:11:14.45-0500 [STG/0] ERR              File "/tmp/contents3962303828/deps/0/python/lib/python3.10/distutils/cmd.py", line 313, in run_command
   2022-02-23T10:11:14.45-0500 [STG/0] ERR                self.distribution.run_command(command)
   2022-02-23T10:11:14.46-0500 [STG/0] ERR              File "/tmp/contents3962303828/deps/0/python/lib/python3.10/distutils/dist.py", line 985, in run_command
   2022-02-23T10:11:14.46-0500 [STG/0] ERR                cmd_obj.run()
   2022-02-23T10:11:14.46-0500 [STG/0] ERR              File "/tmp/contents3962303828/deps/0/python/lib/python3.10/distutils/command/build.py", line 135, in run
   2022-02-23T10:11:14.46-0500 [STG/0] ERR                self.run_command(cmd_name)
   2022-02-23T10:11:14.46-0500 [STG/0] ERR              File "/tmp/contents3962303828/deps/0/python/lib/python3.10/distutils/cmd.py", line 313, in run_command
   2022-02-23T10:11:14.46-0500 [STG/0] ERR                self.distribution.run_command(command)
   2022-02-23T10:11:14.46-0500 [STG/0] ERR              File "/tmp/contents3962303828/deps/0/python/lib/python3.10/distutils/dist.py", line 985, in run_command
   2022-02-23T10:11:14.46-0500 [STG/0] ERR                cmd_obj.run()
   2022-02-23T10:11:14.46-0500 [STG/0] ERR              File "/tmp/contents3962303828/deps/0/python/lib/python3.10/site-packages/setuptools/command/build_py.py", line 56, in run
   2022-02-23T10:11:14.46-0500 [STG/0] ERR                self.build_package_data()
   2022-02-23T10:11:14.46-0500 [STG/0] ERR              File "/tmp/contents3962303828/deps/0/python/lib/python3.10/site-packages/setuptools/command/build_py.py", line 127, in build_package_data
   2022-02-23T10:11:14.46-0500 [STG/0] ERR                srcfile in self.distribution.convert_2to3_doctests):
   2022-02-23T10:11:14.46-0500 [STG/0] ERR            AttributeError: 'Distribution' object has no attribute 'convert_2to3_doctests'
   2022-02-23T10:11:14.46-0500 [STG/0] ERR            ----------------------------------------
   2022-02-23T10:11:14.46-0500 [STG/0] ERR        ERROR: Command errored out with exit status 1
```

**Pull request merged**